cypress/e2e/userIsAbleToConnectAndDisconnectAwsAccount.cy.js

```diff
@@ -37,29 +39,36 @@ describe("Connecting and Disconnecting AWS account", () => {
 		});
 		cy.wait(5000);
 		cy.get('[data-testid="hamburger-menu"]').click();
-
+		cy.wait(5000);
 		cy.get("#root").then($root => {
-			const root = $root.find('[class="amplify-button amplify-field-group__control amplify-button--primary"]');
-			root.length && root[0].innerText === "Sign in"
-				? root[0].click()
-				: cy.get('[data-testid="hamburger-menu"]').click();
+			const asd = $root.find('[class="amplify-button amplify-field-group__control amplify-button--primary"]');
+
+			if (asd.length && asd[0].innerText === "Sign in") {
+				asd[0].click();
+			} else {
+				cy.get('[data-testid="hamburger-menu"]').click();
+			}
 		});
 		cy.wait(5000);
+	});
 
+	it("should allow user to connect, sign-in, sign-out and disconnect AWS account from sidebar", () => {
 		cy.get('[data-testid="hamburger-menu"]').click();
+		cy.get("reach-portal").invoke("remove"); // TODO: remove this after deployment
```

Comment:
Can you elaborate this?

Reply:
@vinguthh this basically remove the hidden DOM element which is being rendered due to the react-spring-bottom-sheet library. This will only be rendered for mobile and tablet devices, not desktop.

Reply:
ic, and we will have a follow PR to proper remove it for desktop as long as adding e2e test for web responsiveness, is it right?